src/Encamina.Enmarcha.SemanticKernel.Connectors.Document/Encamina.Enmarcha.SemanticKernel.Connectors.Document.csproj

```diff
@@ -19,6 +19,7 @@
   </PropertyGroup>
     
   <ItemGroup>
+    <PackageReference Include="ExcelNumberFormat" Version="1.1.0" />
```

Comment:
Is there any other alternative to this package?
It is kind of old (no new updates since 2020), and the owner is not really a known soruce.
That being said, this feature is important, so I will approve these and other changes and create an issue to replace this library for our own implementation (to keep control of it).